frontend/src/components/ViewerPlaceholder.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+
+const ViewerPlaceholder = () => (
```

Comment:
When should this appear? Only when there are no Viewers or also when we have Viewers with 0 instances?
TODO: Also redux logic should remove it on AddViewer action